src/openzaak/selectielijst/tests/test_migrations.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+from zgw_consumers.constants import APITypes
+
+from openzaak.tests.utils import TestMigrations
```

Comment:
Should not be done for this PR but we might want to move define this in `commoground-utils-common`, Open Klant also uses tests like these for example (in an upcoming PR).

Reply:
https://github.com/maykinmedia/commonground-api-common/issues/48